hnn_core/network_models.py

```diff
@@ -185,11 +185,13 @@ def law_2021_model():
     6) Addition of L5_basket -> L5_pyramidal distal connection
     """
 
-    hnn_core_root = op.dirname(hnn_core.__file__)
-    params_fname = op.join(hnn_core_root, 'param', 'default.json')
-    params = read_params(params_fname)
+    if params is not None:
+        hnn_core_root = op.dirname(hnn_core.__file__)
+        params_fname = op.join(hnn_core_root, 'param', 'default.json')
+        params = read_params(params_fname)
```

Comment:
```suggestion
```

This is already handled inside `jones_2009_model`. Should work without these lines